libnet/include/stamp-h.in

```diff
@@ -1 +0,0 @@
-timestamp
```

Comment:
Got me. That's why I removed it! (Seriously, I think it is a remnant from a no-longer-used autotools trick.)

It was added in commit 5ca6de2 which also added a whole lot of random debris, including object files. This file should probably have been deleted when that garbage was cleaned out.